c7n/resources/rds.py

```diff
@@ -117,7 +117,7 @@ class resource_type(object):
     action_registry = actions
     _generate_arn = None
     retry = staticmethod(get_retry(('Throttled',)))
-    permissions = ('rds:ListTagsForResource',)
+    augment = universal_augment
```

Comment:
Any reason why the `rds-snapshot` resource wasn't changed?
It supports universal tagging also:
* http://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/supported-resources.html

I suggest opening a second PR if you decide to do that.

Reply:
yeah definitely a second pr for that, the volume of line change on these with the test data surgery is large.